feature/gribi/mplsutil/mplsutil.go

```diff
@@ -377,23 +377,23 @@ const (
 // type on the ATE device provided.
 func (g *GRIBIMPLSTest) ConfigureFlows(t *testing.T, ate *ondatra.ATEDevice) {
 	t.Helper()
-	switch g.mode {
-	case PushToMPLS:
-		t.Logf("looking on interface %s_ETH for %s", ATESrc.Name, DUTSrc.IPv4)
-		var dstMAC string
-		gnmi.WatchAll(t, ate.OTG(), gnmi.OTG().Interface(ATESrc.Name+"_ETH").Ipv4NeighborAny().LinkLayerAddress().State(), time.Minute, func(val *ygnmi.Value[string]) bool {
-			dstMAC, _ = val.Val()
-			return val.IsPresent()
-		}).Await(t)
-		t.Logf("MAC discovered was %s", dstMAC)
 
-		g.otgConfig.Flows().Clear().Items()
-		mplsFlow := g.otgConfig.Flows().Add().SetName(flowName)
-		mplsFlow.Metrics().SetEnable(true)
-		mplsFlow.TxRx().Port().SetTxName(ATESrc.Name).SetRxName(ATEDst.Name)
+	t.Logf("looking on interface %s_ETH for %s", ATESrc.Name, DUTSrc.IPv4)
+	var dstMAC string
+	gnmi.WatchAll(t, ate.OTG(), gnmi.OTG().Interface(ATESrc.Name+"_ETH").Ipv4NeighborAny().LinkLayerAddress().State(), time.Minute, func(val *ygnmi.Value[string]) bool {
+		dstMAC, _ = val.Val()
+		return val.IsPresent()
+	}).Await(t)
+	t.Logf("MAC discovered was %s", dstMAC)
 
-		mplsFlow.Rate().SetChoice("pps").SetPps(1)
+	g.otgConfig.Flows().Clear().Items()
+	mplsFlow := g.otgConfig.Flows().Add().SetName(flowName)
```

Comment:
Shall we call this something other than `mplsFlow` given that it's just a flow that might not have MPLS?